packages/element/src/react.js

```diff
@@ -185,6 +186,11 @@ export { useRef };
  */
 export { useState };
 
+/**
+ * @see https://reactjs.org/docs/hooks-reference.html#usesyncexternalstore
+ */
+export { useSyncExternalStore };
```

Comment:
Does it make sense to add the rest of the new hooks (`useId`, `useTransition`, `useDeferredValue`, `useInsertionEffect`)? I can immediately see opportunities for almost all of them. 

Note that if we add `useTransition` we might want to expose `startTransition` as well.

Reply:
Good idea, pushed a commit that exports them all.